### PR TITLE
Manage page error message edit

### PR DESCRIPTION
**Pull request by konstantin1390**
Fixes #

### Changes

Summary of changes you made.

* Change 1
* Change 2

### Test plan

Explain how to test changes you made.

* [ ] Step 1
* [ ] Step 2

### Demo (optional)

Sreenshots (before & after) or video that show result of your work.

### CODE REVIEW

File: src/client/rewards/Manage/CampaignRewardsTable/CampaignRewardsTableBodyRow.js [hunk 1]
```diff
@@ -67,14 +83,21 @@ const CampaignRewardsTableRow = ({
...
-        inactivateCampaign(itemOnInactivate, inactivationCampaignData.permlink).then(() => {
-          toggleModal(false);
-          message.success(`Campaign '${currentItem.name}' - has been inactivated`);
-          setLoad(false);
-          setActivationStatus('inactivated');
-        });
+        inactivateCampaign(itemOnInactivate, inactivationCampaignData.permlink)
+          .then(() => {
+            toggleModal(false);
+            message.success(`Campaign '${currentItem.name}' - has been inactivated`);
```
Comment (milklim) [inline src/client/rewards/Manage/CampaignRewardsTable/CampaignRewardsTableBodyRow.js]: add locales

File: src/client/rewards/Manage/CampaignRewardsTable/CampaignRewardsTableBodyRow.js [hunk 2]
```diff
@@ -67,14 +83,21 @@ const CampaignRewardsTableRow = ({
...
+          })
+          .catch(e => {
+            toggleModal(false);
+            message.error(e.error_description);
+            setLoad(false);
+          });
       })
       .catch(() => {
+        toggleModal(false);
         message.error(`Can't inactivate campaign'${currentItem.name}', try again later`);
```
Comment (milklim) [inline src/client/rewards/Manage/CampaignRewardsTable/CampaignRewardsTableBodyRow.js]: use intl

File: src/client/rewards/Manage/CampaignRewardsTable/CampaignRewardsTableBodyRow.js [hunk 3]
```diff
@@ -40,15 +40,31 @@ const CampaignRewardsTableRow = ({
...
-        activateCampaign(currentItem, activationCampaignData.permlink).then(() => {
-          toggleModal(false);
-          message.success(`Campaign '${currentItem.name}' - has been activated`);
-          setLoad(false);
-          setActivationStatus('activated');
-        });
+        activateCampaign(currentItem, activationCampaignData.permlink)
+          .then(() => {
+            toggleModal(false);
+            message.success(`Campaign '${currentItem.name}' - has been activated`);
```
Comment (milklim) [inline src/client/rewards/Manage/CampaignRewardsTable/CampaignRewardsTableBodyRow.js]: use intl.formatMessage

File: src/client/rewards/Manage/CampaignRewardsTable/CampaignRewardsTableBodyRow.js (hunk 1 above)
Comment (konstantin1390) [reply src/client/rewards/Manage/CampaignRewardsTable/CampaignRewardsTableBodyRow.js]: ++

File: src/client/rewards/Manage/CampaignRewardsTable/CampaignRewardsTableBodyRow.js (hunk 2 above)
Comment (konstantin1390) [reply src/client/rewards/Manage/CampaignRewardsTable/CampaignRewardsTableBodyRow.js]: ++

File: src/client/rewards/Manage/CampaignRewardsTable/CampaignRewardsTableBodyRow.js (hunk 3 above)
Comment (konstantin1390) [reply src/client/rewards/Manage/CampaignRewardsTable/CampaignRewardsTableBodyRow.js]: ++

**Pull request merged**